chains/berachain-testnet.json

```diff
@@ -15,11 +15,11 @@
   "providers": [
     {
       "alias": "default",
-      "rpcUrl": "https://bartio.rpc.berachain.com/"
+      "rpcUrl": "https://berachain-testnet-evm-rpc.publicnode.com"
     },
     {
       "alias": "publicnode",
-      "rpcUrl": "https://berachain-testnet-evm-rpc.publicnode.com"
+      "rpcUrl": "https://bartio.rpc.berachain.com"
```

Comment:
Why do we call a non-publicnode provider publicnode? I would call this `official` or something instead.

Reply:
Renamed to `official` which seems more appropriate